hw/top_earlgrey/syn/asic.constraints.sdc

```diff
@@ -181,6 +192,13 @@ set_output_delay ${SPI_DEV_OUT_DEL} [get_ports SPI_DEV_D1]   -clock SPI_DEV_CLK
 set_output_delay ${SPI_DEV_OUT_DEL} [get_ports SPI_DEV_D2]   -clock SPI_DEV_CLK
 set_output_delay ${SPI_DEV_OUT_DEL} [get_ports SPI_DEV_D3]   -clock SPI_DEV_CLK
 
+# False path from CSb to return to host as CSb for that path behaves as reset.
+#set_ideal_network [get_pins top_earlgrey/u_spi_device/u_csb_rst_scan_mux/clk_o]
+set_false_path -through [get_pins top_earlgrey/u_spi_device/cio_csb_i] \
```

Comment:
should the starting point be `-from` here instead of `-through`? 

I do agree that the false path constraint is not ideal, and would also be more in favor of a large max delay or something.
but if this is not a critical path, we can deal with this when working out the backend constraints.

Reply:
`from` requires the port name. I intentionally removed the port in the constraints. Actually I hope to have hierarchical synthesis (+ design constraints too) to make SPI_DEVICE constraints to be SPI_DEVICE only so that I can use `from` and `to` without worrying top connection. :)

And, yes. we can modify this later too.